scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -47,13 +49,32 @@ object AndroidLintReportFormat extends Enumeration {
 @mill.api.experimental
 trait AndroidAppModule extends JavaModule {
 
+  final def root: os.Path = super.millSourcePath
+  private def src: os.Path = root / "src"
+  override def millSourcePath: os.Path = src / "main"
```

Comment:
Let's leave `millSourcePath` unchanged, get rid of `def root`, and adjust the other parts of the code as necessary (e.g. `def sources`). That would make it most similar to how the other Mill modules work

Reply:
thanks, I need to try this out, going for the sources was one of my early attempts , I'm not sure it will work as other things rely on it, but now that I'm more familiar maybe I missed something else

Reply:
done and even spotted a bug with the previous compilation, thanks for your comment!